modules/ROOT/pages/writing.adoc

```diff
@@ -670,8 +738,13 @@ Before the import starts, the code above creates the following schema query:
 CREATE CONSTRAINT FOR (p:Person) REQUIRE (p.surname) IS UNIQUE
 ----
 
-*Take into consideration that the first label is used for the index creation.*
+The name of the created constraint is `spark_NODE_CONSTRAINTS_<LABEL>_<NODE_KEYS>`, where `<LABEL>` is the first label from the `labels` option and `<NODE_KEYS>` is a dash-separated sequence of one or more properties as specified in the `node.keys` options.
+In this example, the name of the created constraint is `spark_NODE_CONSTRAINTS_Person_surname`.
+If the `node.keys` option were set to `"name,surname"` instead, the constraint name would become `spark_NODE_CONSTRAINTS_Person_name-surname`.
 
+The constraint is recreated if it is already present.
```

Comment:
I'm not sure if this is correct. I think we explicitly put `IF NOT EXISTS` clause for constraints.

Reply:
Same here, so shouldn't recreate one.